src/application.ts

```diff
@@ -16,6 +16,9 @@ export class HelloWorldApp extends Application {
     super({
       components: [AuthenticationComponent],
       sequence: MySequence,
+      http: {
+        port: 0,
+      },
```

Comment:
This will start the application on an ephemeral port even when running in production, I don't think that's what a typical user wants.

 - When running in production (typically in a Docker container), the app should listen on a well-known port. Our current default 3000 should be good enough.

 - When running in the test environment, the app needs to listen on port `0`. This can be achieved e.g. by re-binding `http.port` in the test.